dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/createfolder/CreateFolderPresenter.java

```diff
@@ -72,24 +71,19 @@ public CreateFolderPresenter(final Workers workers,
 
     @Override
     public void initialize(final URL location, final ResourceBundle resources) {
-        try {
-            initGUIElements();
-            folderNameField.textProperty().addListener((observable, oldValue, newValue) -> {
-                if (newValue.equals(StringConstants.EMPTY_STRING)) {
-                    createFolderButton.setDisable(true);
-                } else {
-                    createFolderButton.setDisable(false);
-                }
-            });
-            folderNameField.setOnKeyReleased(event -> {
-                if (!createFolderButton.isDisabled() && event.getCode().equals(KeyCode.ENTER)) {
-                    createFolder();
-                }
-            });
-
-        } catch (final Exception e) {
-            LOG.error("Encountered an error making the create folder presenter", e);
-        }
+        initGUIElements();
```

Comment:
Why was this try catch removed?  Without it if there is an exception it will not be caught, and will get swallowed.  It doesn't bubble up into higher layers.

Reply:
There are no exceptions to possibly be thrown in this block that I (or IntelliJ) see